views/AppView.js

```diff
@@ -0,0 +1,64 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .search': 'handleSearch',
+    'click .view-video': 'viewVideo'
+  },
+
+  mainVideo: null,
+
+  initialize: function () {
+    this.listenTo(this.model.get('videos'), 'add', this.renderVideo);
```

Comment:
Instead of repeating this code, you could write:

```js
this.listenTo(this.model.get('videos'), 'add', function () {
  this.renderVideo();
  this.setCurrentVideo();
});
```